src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -134,16 +134,16 @@
         trim_chars=("@",),
     ),
     # Add a Full equivalent which also allow keywords
-    NakedIdentifierSegmentFull=RegexParser(
+    NakedIdentifierFullSegment=RegexParser(
```

Comment:
Why no `Grammar` suffix here?

Reply:
Seems like all
The Identifiers are set up as Segments rather than Grammars. Which makes sense to me since they do get a name and a type. Do you disagree?

Btw there’s a few though that don’t do exactly this (e.g. `ParameterNameSegment` in BigQuery is a `OneOf` of two RegexParsers) but left that as was.

Reply:
No strong feelings here, just kind of mechanically looking over the PR. Moving `Segment` to the end of the name seems like an improvement, regardless.